backend/src/DatabaseHandler.py

```diff
@@ -0,0 +1,123 @@
+# General imports =================================================================================
+import datetime
+import json
+import time, os, sys, git
+import multiprocessing as mp
+from pocketbase import Client
+from pocketbase.services.realtime_service import MessageData
+
+# Project specific imports ========================================================================
+from src.support.CommonLogger import logger
+
+from src.Utils import Utils as utl, THREAD_MESSAGE_DB_WRITE, THREAD_MESSAGE_KILL, THREAD_MESSAGE_SERIAL_WRITE, WorkQ_Message
+
+# Class Definitions ===============================================================================
+class DatabaseHandler():
+    def __init__(self,thread_name: str, thread_workq: mp.Queue, message_handler_workq: mp.Queue):
+        """
+        Thread to handle the pocketbase database communication.
+        The Thread is subscribed to the CommandMessage
+        collection to wait for commands created in the front end. 
+        The handler can also send telemetry data to the database
+        to be read by the front end.
+        """
+        logger.info("DatabaseHandler initializing")
+        DatabaseHandler.thread_workq = thread_workq
+        DatabaseHandler.send_message_workq = message_handler_workq
+        DatabaseHandler.thread_name = thread_name
+        DatabaseHandler.client = Client("http://10.13.141.121:8090/")
+        DatabaseHandler.client.collection('CommandMessage').subscribe(DatabaseHandler._handle_command_callback)
```

Comment:
Need to make sure this is the correct ip.

Reply:
what is the correct IP? i added a thing to the readme to make sure this is generalized for front end and backend but If you have to IP I can put it in now

Reply:
We don't know yet. We'll have to setup the soar router and connect and see. This was just a todo/make sure we update it at some point.

Reply:
sounsd good its ddone